File: K번째수.java
import java.util.*;
class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];
        
        for(int i=0; i< commands.length; i++){
                int start = commands[i][0];
                int end = commands[i][1];
                int k = commands[i][2];
            
                int[] tmp=new int[end-start+1];
                int a=0;
                for(int j=start-1; j<end; j++){
                    tmp[a++]=array[j];
                }
                Arrays.sort(tmp);
                answer[i]=tmp[k-1];
        }
        
        return answer;
    }
}

File: K번째 수.js
function solution(array, commands) {
    var answer = [];
    var real_answer = [];
   for(var i = 0; i < commands.length; i++)
       {
            answer = array.slice(commands[i][0]-1,commands[i][1]);
          
           //정렬
           answer.sort(function(a, b) {
                return a - b;
            });
            real_answer.push(answer[commands[i][2]-1]);
       }   
    return real_answer;   
}

console.log(solution([1, 5, 2, 6, 3, 7, 4],[[2, 5, 3], [4, 4, 1], [1, 7, 3]]
));